.azure/templates/build-config-user.yml

```diff
@@ -22,6 +22,13 @@ jobs:
     ${{ if eq(parameters.platform, 'windows') }}:
       path: msquic
 
+  - task: UseDotNet@2
```

Comment:
This adds nearly 2 minutes to our build time. Is it really necessary?

Reply:
I can disable it on windows. It is required for linux, where it only takes 6 seconds...

Reply:
Please do so.

Reply:
Done